pages/Product/components/Reviews/components/List/components/Review/components/Rating/index.jsx

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) 2017, Shopgate, Inc. All rights reserved.
+ *
+ * This source code is licensed under the Apache 2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import I18n from '@shopgate/pwa-common/components/I18n';
+import RatingStars from '../../../../../../../../../../components/RatingStars';
+import style from './style';
+
+/**
+ * Review Rating Component
+ * @param {number} rate The rating value
+ * @constructor
+ */
+const Rating = ({ rate }) => {
+  let stars = 1;
```

Comment:
This should be written in shorthand.
`const stars = (rate > 20) ? (rate / 20) : 1;`

Reply:
Done.